<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(32, 32, 32, 180)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>24</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Steganography Tool&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>131</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="mode_readBtn">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Read Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode_writeBtn">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Write Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mode_AboutBtn">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>99</y>
      <width>871</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QPushButton" name="exec_Read">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Read Data From the Image.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Output Text File:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="output_TxtFile">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="accessibleDescription">
          <string notr="true"/>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="input_ImageDir">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Input Image:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Data length:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Input Text File:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Input Image:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="input_TxtFile">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="exec_Write">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Write Data to the Image.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="textBrowser_9">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextBrowser" name="textBrowser_10">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_23">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>Output Directory:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="input_ImageDir1">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextBrowser" name="textBrowser_11">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255,255,255)</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="output_ImageDir">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(64,64,64);color:rgb(255,255,255)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>491</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Author: HaKai&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;GitHub: https://github.com/HaKai-2062&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Note: Currently Only Supports 24bit Bitmaps&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
